packages/mosip-api/src/index.ts

```diff
@@ -29,9 +30,18 @@ const envToLogger = {
 const app = Fastify({
   logger: envToLogger[env.isProd ? "production" : "development"],
 });
+let whitelist: string[] = [env.DOMAIN]
+if (env.DOMAIN[0] !== '*') {
+  whitelist = [env.CLIENT_APP_URL, env.COUNTRY_CONFIG_URL]
```

Comment:
@naftis what other URLs should be whitelisted?

Reply:
@euanmillar I think we only need to whitelist client as CORS only applies frontend-to-backend

Reply:
```suggestion
  whitelist = [env.CLIENT_APP_URL]
```
@naftis so should it be like this?